Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3884,6 +3884,18 @@ InspectorAttachmentSide:
     WebKit:
       default: 0
 
+InspectorMaximumResourcesContentSize:
+  type: uint32_t
+  status: embedder
+  inspectorOverride: true
```

Comment:
please remove this as `inspectorOverride` is used to allow Web Inspector to override this setting from the UI (i.e. it's not a category for "this setting is used by Web Inspector") <[https://webkit.org/web-inspector/device-settings/](https://webkit.org/web-inspector/device-settings/)>